.github/workflows/ci.yml

```diff
@@ -49,6 +49,18 @@ jobs:
       - name: check namespacing
         run: |
           ./scripts/ci/check-namespace
+  quickcheck-windows:
+    name: Quickcheck windows-latest
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - uses: ilammy/msvc-dev-cmd@v1
```

Comment:
How configurable is this? Can we test different MSVC versions?

Reply:
We seemed to be able to test a bunch of different set of MSVC configuration

https://github.com/ilammy/msvc-dev-cmd?tab=readme-ov-file#inputs

> arch – target architecture
> - native compilation:
>   - x64 (default) or its synonyms: amd64, win64, x86_64, x86-64
>   - x86 or its synonyms: win32
> cross-compilation: x86_amd64, x86_arm, x86_arm64, amd64_x86, amd64_arm, amd64_arm64
> sdk – Windows SDK to use
> - do not specify to use the default SDK
> - or specify full Windows 10 SDK number (e.g, 10.0.10240.0)
> - or write 8.1 to use Windows 8.1 SDK
> toolset – select VC++ compiler toolset version
> - do not specify to use the default toolset
> - 14.0 for VC++ 2015 Compiler Toolset
> - 14.XX for the latest 14.XX toolset installed (e.g, 14.11)
> - 14.XX.YYYYY for a specific full version number (e.g, 14.11.25503)
> uwp – set true to build for Universal Windows Platform (i.e., for Windows Store)
> spectre – set true to use Visual Studio libraries with [Spectre](https://meltdownattack.com/) mitigations
> vsversion – the Visual Studio version to use. This can be the version number (e.g. 16.0 for 2019) or the year (e.g. "2019"); omit this input to select the latest version of Visual Studio. On [GitHub-hosted runners](https://docs.github.com/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners), this input is not required because there is only one Visual Studio in the environment.

Reply:
Great! Could you make it so the CI tests with an older version as well?